src/core/inline-idl-parser.js

```diff
@@ -132,6 +132,7 @@ function renderEnum(details) {
     data-xref-type="enum-value"
     data-link-for="${identifier}"
     data-xref-for="${identifier}"
+    data-lt="${!enumValue ? "the-empty-string" : null}"
```

Comment:
Is this Shepherd compatible? What does Shepherd to for this?

Reply:
Good catch. It isn't.
Need to convert `the-empty-string` back to `""` in core/xref. 
For local linking, need to keep using `the-empty-string`, right? (I used this as we were already using it until 57d9bbc).